### PR TITLE
translations/transactions

### PR DESCRIPTION
**Pull request by vyvil**
PR to apply translations to transactions related screens

### CODE REVIEW

File: src/containers/SendContainer/SendContainer.js [hunk 1]
```diff
@@ -100,8 +101,8 @@ const SendContainer = (props: Props) => {
         {...customSelectorProps}
       />
       <ValueSelectorCard
-        selectorModalTitle="Send"
-        maxLabel="Send max"
+        selectorModalTitle={t('transactions.title.valueSelectorModal')}
+        maxLabel={t('button.sendMax')}
```
Comment (wjkg) [inline src/containers/SendContainer/SendContainer.js:105]: `button.sendMax` is missing in the translations file.

Comment (vyvil) [reply src/containers/SendContainer/SendContainer.js:105]: Thanks! Added :)

File: src/screens/Tank/SettleBalanceConfirm/SettleBalanceConfirm.js [hunk 2]
```diff
@@ -162,11 +162,11 @@ class SettleBalanceConfirm extends React.Component<Props, State> {
 
     const reviewData = [
       {
-        label: 'Assets to settle',
+        label: t('transactions.label.assetsToSettle'),
         valueArray: txToSettle,
       },
       {
-        label: 'Transaction fee',
+        label: t('transactions.label.transactionFee'),
```
Comment (wjkg) [inline src/screens/Tank/SettleBalanceConfirm/SettleBalanceConfirm.js:169]: Looks intentional, but just in case, this string is set to "Est. Network Fee".

Comment (vyvil) [reply src/screens/Tank/SettleBalanceConfirm/SettleBalanceConfirm.js:169]: Yeah, I've unified those :)

File: src/translations/locales/en/common.json [hunk 3]
```diff
@@ -293,7 +340,12 @@
   "error": {
     "transactionFailed": {
       "timeOut": "Transaction timed out",
-      "default": "Transaction failed"
+      "default": "Transaction failed",
+      "offline": "You can't send transaction while offline",
+      "notEnoughGas": "Not enough gas to cover the transaction fee. Top up your ETH balance",
+      "notOwnedCollectible": "You do not longer own this collectible",
+      "collectibleCantBeTransferred": "This collectible can not be transferred",
```
Comment (wjkg) [inline src/translations/locales/en/common.json:348]: If this is a good moment for corrections, these two messages don't look right to me. "do no longer own" and "can't/cannot be transferred"?

**Review by wjkg — APPROVED** [review]
LGTM

**Review by poocart — APPROVED** [review]

File: src/translations/locales/en/common.json (hunk 3 above)
Comment (vyvil) [reply src/translations/locales/en/common.json:348]: Oh, true. Didn't notice those, to be honest ;/
Will adjust those. Thanks!

**Pull request merged**